.github/workflows/build.yaml

```diff
@@ -35,3 +39,4 @@ jobs:
       - name: see test coverage
```

Comment:
Theres no need to run both the `yarn test` script and the `yarn coverage` script. They will both run the tests, but the coverage version will also check coverage.

The `yarn test` script will also start in watch mode (but may have some auto detection that it is running in CI env so it wont cause the build to hang), or this may be the reason its hanging.

I would suggest to remove the `run tests` task (line 35-37).